samples/echo/proguard-rules.pro

```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
```

Comment:
Do we need this file?

Reply:
not really as it is empty: studio generated template file. deleted